mappings/net/minecraft/entity/player/InventoryPlayer.mapping

```diff
@@ -41,14 +41,21 @@ CLASS none/aal net/minecraft/entity/player/InventoryPlayer
 		ARG 0 stack
 	METHOD c_ getInvProperty (I)I
 		ARG 0 pos
+	METHOD d removeOne (Lnone/aes;)V
+		ARG 0 item
+	METHOD e validHotbarIndex (I)Z
```

Comment:
isValidHotbarIndex.

(as a future note, please try to use is/does/can/has for boolean methods and *NOT* use it for boolean fields)